### ISSUE TITLE
Narrator does not announce descriptively when focus is on breadcrumb

### ISSUE THREAD

**Issue by yueou1995** [microsoft/fluentui#15531]
### Environment Information

- Windows Version - Win10
- Chromium Edge - Version 85.0.564.51 (Official build) (64-bit)

### Describe the issue:

Narrator does not announce descriptively when focus is on breadcrumb.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/yueou/pen/BazjMZX

#### Actual behavior:

Screen reader reads 'navigation landmark ...'

#### Expected behavior:

Screen reader should read 'navigation landmark **breadcrumb** ...'

### Documentation describing expected behavior

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b248054a6-5e68-4771-9e1e-242fb5025730%7d

**Comment by paulgildea** [comment]
@yueou1995 Thanks for the issue.

@smhigley FYI

**Comment by tomi-msft** [comment]
Hi @yueou1995 thanks for filing the issue. I'm having trouble reproing the behavior given the steps that you've outlined. Could you provide a bit more information? 

Something else to note is that there needs to be an ariaLabel prop for the screen reader to read Breadcrumb. This is present in the codepen, so it could be that is why I am unable to repro the issue.
